sktm/executable.py

```diff
@@ -220,11 +228,33 @@ def load_config(args):
     return cfg
 
 
+@contextmanager
+def ensure_single_instance(filename):
+    lock_file = os.path.join('/tmp', filename)
+    fp = open(lock_file, 'w')
+    try:
+        fcntl.lockf(fp, fcntl.LOCK_EX | fcntl.LOCK_NB)
+    except IOError:
+        sys.exit("There is another instance already running")
```

Comment:
Could you please check that the IOError has errno of EACCESS or EAGAIN, and only then do the "exit" with the message. Otherwise, please re-raise the original exception. This will prevent confusion in the case of another failure.